### PR TITLE
[runTheMatrix.py] Adding cleanedupgrade matrix for PR tests

### PR DESCRIPTION
**Pull request by silviodonato**
#### PR description:

This PR add a new matrix in runTheMatrix.py that is defined as a copy of the `upgrade` matrix, after having removed all the workflows in common to the other matrices (eg. `23461.97` is both is `2026` and `upgrade`).
This PR will allow the PR tests to run `cleanedupgrade` together with the other matrices without problems. In other words, this PR will solve the problems of the duplicated wfs (see  #33338 and #33732)

#### PR validation:

```
runTheMatrix.py  -l 38132.98,10024.0  --what standard,highstats,pileup,generator,production,ged,2017,2026,machine,cleanedupgrade
```

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

No backport needed

### CODE REVIEW

**Comment by silviodonato** [comment]
test parameters:
  - relvals_options = --what standard,highstats,pileup,generator,production,ged,2017,2026,machine,premix,cleanedupgrade

**Comment by silviodonato** [comment]
please test workflow 37725.0

**Comment by silviodonato** [comment]
btw. I think this solution was proposed by @fwyzard during a ORP meeting long time ago.

**Comment by silviodonato** [comment]
please abort

**Comment by silviodonato** [comment]
please test workflow 37725.0

**Comment by silviodonato** [comment]
please test workflow 36216.0

File: Configuration/PyReleaseValidation/python/relval_cleanedupgrade.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+from Configuration.PyReleaseValidation.relval_ged import workflows as _ged
+from Configuration.PyReleaseValidation.relval_gpu import workflows as _gpu
+from Configuration.PyReleaseValidation.relval_2017 import workflows as _2017
+from Configuration.PyReleaseValidation.relval_2026 import workflows as _2026
+from Configuration.PyReleaseValidation.relval_machine import workflows as _machine
+from Configuration.PyReleaseValidation.relval_premix import workflows as _premix
+
+from Configuration.PyReleaseValidation.relval_upgrade import workflows as _upgrade_workflows
+
+wfAlreadyIncluded = []
```
Comment (srimanob) [inline Configuration/PyReleaseValidation/python/relval_cleanedupgrade.py:23]: I don't see where this is used.

Comment (silviodonato) [reply Configuration/PyReleaseValidation/python/relval_cleanedupgrade.py:23]: good point. I'm fixing it

**Comment by silviodonato** [comment]
please test workflow 36216.0

**Comment by srimanob** [comment]
+Upgrade

**Comment by chayanit** [comment]
+1

**Comment by silviodonato** [comment]
+1

**Pull request merged**